infrastructure/kube/keep-dev/relay-maintainer-deployment.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: relay-maintainer
+  namespace: default
+  labels:
+    keel.sh/policy: all
+    app: maintainer
+    type: relay
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: maintainer
+      type: relay
+  template:
+    metadata:
+      labels:
+        app: maintainer
+        type: relay
+    spec:
+
+      initContainers:
+        - name: initcontainer-provision-relay-maintainer
+          image: gcr.io/keep-dev-fe24/initcontainer-provision-relay-maintainer
+          imagePullPolicy: Always
+          env:
+
+            - name: ETH_RPC_URL
+              valueFrom:
+                configMapKeyRef:
+                  name: eth-network-internal
+                  key: rpc-url
+
+            - name: ETH_NETWORK_ID
+              valueFrom:
+                configMapKeyRef:
+                  name: eth-network-internal
+                  key: network-id
+
+            - name: ETH_NETWORK_NAME
+              value: internal
+
+            - name: RELAY_MAINTAINER_ETH_ACCOUNT_ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: eth-account-info
+                  key: account-0-address
+
+            - name: RELAY_MAINTAINER_ETH_ACCOUNT_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: eth-account-privatekeys
+                  key: account-0
+
+            - name: CONTRACT_OWNER_ETH_ACCOUNT_ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: eth-network-internal
+                  key: contract-owner-eth-account-address
+
+            - name: CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: eth-network-internal
+                  key: contract-owner-eth-account-private-key
+
+            - name: BCOIN_HOST
+              value: bcoin.default.svc.cluster.local
+
+            - name: BCOIN_PORT
+              value: '18332'
+
+            - name: BCOIN_API_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: bcoin
+                  key: http-api-key
+
+          volumeMounts:
+            - name: relay-maintainer-env
+              mountPath: /mnt/relay-maintainer
+          command: ['node', '/tmp/provision-relay-maintainer.js']
```

Comment:
The `Dockerfile` already sets an appropriate entry point, yah?

Reply:
I go back and forth on this, it feels good to be explicit on the app config with what command is being run for a container, even if it's the default entry point.

Reply:
I feel like this is probably true for external containers, since they can change out from under you, but for containers we control it feels like we're just doubling our workload whenever we need to change something… Either way, it's fine for this PR, especially if it's what we've been doing elsewhere.